docs/DeveloperGuide.adoc

```diff
@@ -569,47 +569,58 @@ the module database up to date, or to cater to their own needs.
 [NOTE]
 The commands pertaining to module database administration can only be executed by users of type `Admin`.
 
+Administrators can choose to add, remove or edit a module using `addModuleDB`, `removeModuleDB` and `editModule` respectively.
+The changes will then be reflected
+in both the module list in memory and in storage. Figure 1(shown below) describes the general flow of the administration
+of the module database in an activity diagram.
+
+image::ADforModuleDatabaseAdministration.png[width="800]
```

Comment:
It would be good to include a description on what is contained moduleList.xml

Reply:
Added a note mentioning that module database is stored in moduleList.xml